week3/homework/src/index.js

```diff
@@ -13,35 +13,34 @@ const editToDo = require('./editToDo');
 app.use(express.json());
 app.use(bodyParser.json());
 
-app.post('/addTodo', (req, res) => {
-  const toWrite = req.body.todo.description;
+app.post('/creatingNewTodo', (req, res) => {
```

Comment:
The route should be `app.post('/todo', (req, res) => { ... })`

Reply:
@gajduk  Sorry just want to ask you if you can see the the changes or not ?